ee/tables/tablewrapper/tablewrapper.go

```diff
@@ -0,0 +1,67 @@
+package tablewrapper
+
+import (
+	"context"
+	"fmt"
+	"log/slog"
+	"time"
+
+	"github.com/kolide/launcher/ee/gowrapper"
+	"github.com/kolide/launcher/pkg/traces"
+	"github.com/osquery/osquery-go/plugin/table"
+)
+
+const DefaultTableTimeout = 2 * time.Minute
+
+type generateResult struct {
+	rows []map[string]string
+	err  error
+}
+
+// NewTablePluginWithTimeout returns a table plugin using the default table timeout.
+func NewTablePluginWithTimeout(slogger *slog.Logger, name string, columns []table.ColumnDefinition, gen table.GenerateFunc) *table.Plugin {
```

Comment:
This signature feels wrong. Feels like it should just be `New(...)` with a functional argument to override the default timeout